ftw/slider/upgrades/20170824081802_add_i_name_from_title_behavior_to_ftw_slider_pane/types/ftw.slider.Pane.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<object name="ftw.slider.Pane">
+
+    <property name="behaviors" purge="False">
+        <element value="plone.app.content.interfaces.INameFromTitle"/>
```

Comment:
Does this ensure uniqueness? 🤔 

The fix of the old upgrade step makes that the `INameFromTitle` behavior is no longer removed in the old upgrade, but it is still added here. Will it then end up twice in the behaviors list? 🤔